src/file-selector.spec.ts

```diff
@@ -1,18 +1,13 @@
 import { FileWithPath } from "./file.js";
-import { fromEvent } from "./file-selector.js";
+import { fromEvent, fromFileHandles } from "./file-selector.js";
 
 it("returns a Promise", async () => {
   const evt = new Event("test");
   expect(fromEvent(evt)).toBeInstanceOf(Promise);
 });
 
 it("should return an empty array if the passed arg is not what we expect", async () => {
-  const files = await fromEvent({});
-  expect(files).toHaveLength(0);
-});
-
-it("should return an empty array if drag event", async () => {
```

Comment:
I removed this test, as it was doing the exact same thing the test above is doing.